ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXHeightMapGraphicNew.java

```diff
@@ -132,7 +133,7 @@ private static List<RDXMultiColorMeshBuilder> generateHeightCells(IntToDoubleFun
             {
                double x = HeightMapTools.indexToCoordinate(xIndex, gridCenterX, gridResolutionXY, centerIndex);
                double y = HeightMapTools.indexToCoordinate(yIndex, gridCenterY, gridResolutionXY, centerIndex);
-               vertices[xIndex][yIndex] = new Point3D(x, y, groundHeight);
+               vertices[xIndex][yIndex] = new Point3D(x, y, 0.0);
```

Comment:
Ooooooo I remember that we did this, remember the groundHeight was NaN so we changed it to 0.0 to see if that would be a problem. We probably don't wanna leave it like that.
Is there a way we can send default ground height? We may want to add this to the message `heightMapMessage.getEstimatedGroundHeight()` or get access to it so we can use it rather then doing 0.0